cogs5e/initTracker.py

```diff
@@ -381,30 +384,33 @@ async def nextInit(self, ctx):
             await ctx.send("It is not your turn.")
             return
 
-        toRemove = []
-        if combat.current_combatant is not None:
+        advanced_round, messages = combat.advance_turn()
```

Comment:
This should probably still be after the deathdelete code, since that code checks `combat.current_combatant`, which is updated by `advance_turn`